Add an Italian translation

alexinfurs opened the pull request.

alexinfurs committed:
Update CATSFC/system/language.msg

Add an Italian translation

alexinfurs committed:
Update source/nds/message.h

Add Italian language

alexinfurs committed:
Update source/nds/gui.c

Add Italian language

Nebuleon commented:
Whoops! I didn't see this request before making my latest option. Could you please add a new commit with these text strings:

#MSG_HOTKEY_QUICK_LOAD_STATE
Load saved state 1
#MSG_HOTKEY_QUICK_SAVE_STATE
Create saved state 1
#MSG_HOTKEY_FULL_SCREEN_TOGGLE
Toggle full-screen

below this text:

#MSG_HOTKEY_SOUND_TOGGLE
Attiva/Disattiva audio

"Toggle full-screen" is a hotkey that changes the display mode from [3] Middle of the screen to [4] Entire screen, smoothed, or back from 4 to 3 when used.

---

You're also missing the "retro sound" option:

#FMT_AUDIO_RETRO_SOUND
Retro sound       %s

below this one:

#FMT_AUDIO_SOUND
Sound                  %s

"Retro sound" is a way for the SNES emulation to sound like it's a retro NES instead.

Nebuleon commented:
Some option values have text that is too long when used in the menu:

Modalità di visualizzazione [4] Schemo intero,
Lisciato

This option name and value wraps to the next line. It also doesn't align well in two columns like in English due to the length of "Modalità di visualizzazione".

In the Tools/Hotkeys menus (tasti di scelta rapida ...), "Avanzamento rapido temporaneo" goes onto the hotkey button display, so if I set it to R, it looks like "Avanzamento rapido tempo■neo". Is it acceptable to use "Avanzamento rapido temp."?

alexinfurs committed:
Update CATSFC/system/language.msg

alexinfurs committed:
Update CATSFC/system/language.msg

alexinfurs commented:
Ok, I've added the missing options.

I've changed "Modalità di visualizzazione" to "Modalità video". Now I think it has the right length.

I've also changed "Avanzamento rapido temporaneo" to "Avanzamento rapido temp." as you suggested. 

Also I've changed "Lisciato" to "Smussato", just because the first one sounds a little odd.
